app_data/vocabularies/resource_types.yaml

```
@@ -0,0 +1,1057 @@
+    datacite_general: Text
+    datacite_type: Project milestone
+    openaire_resourceType: "0035"
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/report
+    schema.org: https://schema.org/CreativeWork
+    subtype: publication-milestone
+    type: publication
+    marc21_type: publication
+    marc21_subtype: milestone
+  title:
+    en: Project milestone
+    sv: Milstolpe för projektet
+    de: Projektmeilenstein
+    cs: Mezník projektu
+  tags:
+    - depositable
+    - linkable
+- id: publication-proposal
+  icon: file alternate
+  props:
+    csl: article
+    datacite_general: Text
+    datacite_type: Proposal
+    openaire_resourceType: "0036"
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/researchProposal
+    schema.org: https://schema.org/CreativeWork
+    subtype: publication-proposal
+    type: publication
+    marc21_type: publication
+    marc21_subtype: proposal
+  title:
+    en: Proposal
+    sv: Förslag
+    de: Antrag
+    cs: Návrh
+  tags:
+    - depositable
+    - linkable
+- id: publication-report
+  icon: file alternate
+  props:
+    csl: article
+    datacite_general: Report
+    datacite_type: ""
+    openaire_resourceType: "0017"
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/report
+    schema.org: https://schema.org/ScholarlyArticle
+    subtype: publication-report
+    type: publication
+    marc21_type: publication
+    marc21_subtype: report
+  title:
+    en: Report
+    sv: Rapport
+    de: Bericht
+    cs: Zpráva
+  tags:
+    - depositable
+    - linkable
+- id: publication-softwaredocumentation
+  icon: file alternate
+  props:
+    csl: article
+    datacite_general: Text
+    datacite_type: Software documentation
+    openaire_resourceType: "0009"
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/technicalDocumentation
+    schema.org: https://schema.org/CreativeWork
+    subtype: publication-softwaredocumentation
+    type: publication
+    marc21_type: publication
+    marc21_subtype: softwaredocumentation
+  title:
+    en: Software documentation
+    sv: Mjukvarudokumentation
+    de: Softwaredokumentation
+    cs: Dokumentace softwaru
+  tags:
+    - depositable
+    - linkable
+- id: publication-technicalnote
+  icon: file alternate
+  props:
+    csl: article
+    datacite_general: Text
+    datacite_type: Technical note
+    openaire_resourceType: "0009"
+    openaire_type: publication
+    eurepo: info:eu-repo/semantics/technicalDocumentation
+    schema.org: https://schema.org/ScholarlyArticle
+    subtype: publication-technicalnote
+    type: publication
+    marc21_type: publication
+    marc21_subtype: technicalnote
+  title:
+    en: Technical note
+    sv: Technical note
+    de: Technical Note
+    cs: Technická poznámka
+  tags:
+    - depositable
+    - linkable
+- id: publication-thesis
+  icon: file alternate
+  props:
+    csl: thesis
+    datacite_general: Text
+    datacite_type: Thesis
```

Didn't we recently apply this change? See [here](https://datacite-metadata-schema.readthedocs.io/en/4.6/appendices/appendix-1/resourceTypeGeneral/#dissertation)
```suggestion
    datacite_general: Dissertation

```

Below we have dissertation. Please double-check